docs/samples/README.md

```diff
@@ -22,6 +24,46 @@ __Note:__ Mira accepts both `.js` and `.ts` files as the entry file. Mira will l
 
 Follow the `README.md` file provided in the sample app for full instructions.
 
+## Titus App
+
+- [Titus](https://github.com/nearform/titus)
+
+Titus is a development and deployment stack for SaaS applications that encapsulates best practice and supports rapid innovation.
+
+Using Titus across the organisation drives consistency while accelerating project set-up and deployment. Even unskilled cloud developers can rapidly get from project kick-off to feature development without understanding all of the underlying cloud services.
+
+Titus provides excellent developer experience throughout projects and allows rapid on-boarding of new developers.
+
+
+### Overview
+This sample application uses many AWS services and can be used as a reference to create a complex infrastructure using Mira.
+
+An high level description of TITUS can be summarized with:
+
+- React frontend
+- Api service
+- Authentication with Cognito
+- Data storage on RDS Postgres
```

Comment:
Can we have a description of what and why data storage is needed? (the other 3 elements are all covered below, but not this one)